helix-term/src/application.rs

```diff
@@ -372,6 +400,17 @@ impl Application {
                     MethodCall::WorkDoneProgressCreate(params) => {
                         self.lsp_progress.create(server_id, params.token);
 
+                        let ui = self
+                            .compositor
+                            .find(std::any::type_name::<ui::EditorView>())
+                            .unwrap();
+                        if let Some(ui) = ui.as_any_mut().downcast_mut::<ui::EditorView>() {
```

Comment:
Should be safe to simply unwrap/expect since it's guaranteed we found the `EditorView`

Reply:
I added a expect and unwrap to reflect that.